app/main.py

```diff
@@ -1,8 +1,26 @@
 class Person:
-    # write your code here
-    pass
 
+    people = {}
```

Comment:
The `people` dictionary is a class attribute, which means it is shared across all instances of `Person`. If you intend to manage people separately for different contexts, consider using an instance attribute or another mechanism to store this information.